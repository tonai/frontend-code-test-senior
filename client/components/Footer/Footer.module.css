.footer {
  background: var(--hemocyanin);
  color: var(--purpleHaze);
  margin: 0 calc(var(--sideMargin) * -1);
  padding: 25px var(--sideMargin);
  font-size: 11px;
  line-height: 18px;
}
