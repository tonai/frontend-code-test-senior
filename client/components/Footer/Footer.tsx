import css from "./Footer.module.css";

export default function Footer() {
  return (
    <footer className={css.footer}>
      <p>Octopus Energy Ltd is a company registered in England and Wales.</p>
      <p>
        Registered number: 09263424. Registered office: 33 Holborn, London, EC1N
        2HT. Trading Office: 20-24 Broadwick Street, London, W1F 8HT
      </p>
    </footer>
  );
}
