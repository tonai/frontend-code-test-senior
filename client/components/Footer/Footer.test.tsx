import { render } from "@testing-library/react";

import Footer from "./Footer";

describe("Footer component", () => {
  it("matches snapshot", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
