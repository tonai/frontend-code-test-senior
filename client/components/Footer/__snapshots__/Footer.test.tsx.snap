// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<div>
  <footer
    class="footer"
  >
    <p>
      Octopus Energy Ltd is a company registered in England and Wales.
    </p>
    <p>
      Registered number: 09263424. Registered office: 33 Holborn, London, EC1N 2HT. Trading Office: 20-24 Broadwick Street, London, W1F 8HT
    </p>
  </footer>
</div>
`;
